celavi/routing.py

```diff
@@ -99,6 +116,10 @@ def get_route(self, start, end):
             .sum().reset_index()
 
         _summary['region_transportation'] = _summary['statefp'] + _summary['countyfp']
+        # TODO: Consider generalizing below. An input parameter should be used.
+        #  At least 0.621371 should be stored in a variable such as
+        #  mile_per_km (I suppose it's the conversion factor mile-km).
+        #  A comment should also specify why weight is divided by 1000.
```

Comment:
The mile-to-km conversion TODO item will be resolved with issue #108 as it also affects CostGraph code